pkg/datasource/columns.go

```diff
@@ -262,6 +267,9 @@ var annotationsTemplates = map[string]map[string]string{
 		ColumnsMinWidthAnnotation:  "8",
 		ColumnsAlignmentAnnotation: string(metadatav1.AlignmentRight),
 	},
+	"address": {
+		ColumnsHexAnnotation: "true",
+	},
```

Comment:
I think it's too generic. It can be a memory address, network address, etc. IMO we don't need a template for this, we can set it each field that wants to use this feature, like the address field of the trace_malloc gadget: 

https://github.com/inspektor-gadget/inspektor-gadget/blob/f443360a9f8261801c4444a8b45e7ac94289f25f/gadgets/trace_malloc/gadget.yaml#L45-L48